src/app/features/todos/todos-footer/todos-footer.component.html

```diff
@@ -0,0 +1,11 @@
+<footer class="todos-footer">
+  <span class="todos-footer-col actor">
+    <button id="clearCompleted" class="guardian" (click)="handleFilter($event)">Clear completed</button>
```

Comment:
Rank B
You should use HTML attribute `data-*`

Reply:
I do not understand what you mean.